code/game/objects/items/weapons/storage/briefcase.dm

```diff
@@ -78,6 +78,11 @@
 	var/bottom_open = 0 //is the false bottom open?
 	var/obj/item/stored_item = null //what's in the false bottom. If it's a gun, we can fire it
 
+/obj/item/weapon/storage/briefcase/false_bottomed/examine(mob/user)
+	..()
+	if(user.is_holding_item(src))
+		to_chat(user, "This one feels a bit heavier than normal for how much fits in it.")
```

Comment:
add a font here bro

Reply:
you know I thought it did look a little funny but since you pointed it out you get to choose the font Barry